docs/src/components/home/Testimonials.tsx

```diff
@@ -0,0 +1,183 @@
+  { title: '69k', metadata: 'Stars on GitHub' },
+  { title: '2k', metadata: 'Open-source contributors' },
+  { title: '13k', metadata: 'Followers on Twitter' },
+];
+
+const UserFeedback = ({
+  quote,
+  profile,
+}: {
+  quote: string;
+  profile: { avatar: string; name: string; role: string; company?: React.ReactElement };
+}) => {
+  return (
+    <Box>
+      <Typography variant="subtitle1" color="#fff" sx={{ mb: 2 }}>
+        {quote}
+      </Typography>
+      <Box sx={{ display: 'flex' }}>
+        <Avatar
+          src={profile.avatar}
+          alt={`Picture of ${profile.name}`}
+          sx={{ width: 60, height: 60, border: '1px solid', borderColor: '#fff' }}
+        />
+        <Box sx={{ ml: 2 }}>
+          <Typography color="#fff" fontWeight="bold" sx={{ mb: 1 }}>
+            {profile.name},{' '}
+            <Box component="span" sx={{ color: 'grey.500', fontWeight: 'regular' }}>
+              {profile.role}
+            </Box>
+          </Typography>
+          {profile.company}
+        </Box>
+      </Box>
+    </Box>
+  );
+};
+
+const TESTIMONIALS = [
+  {
+    quote:
+      "“Material-UI offers a wide variety of high quality components that have allowed us to ship features faster. Material-UI has been used by more than a hundred engineers in our organization. What’s more, Material-UI's well architected customization system has allowed us to differentiate ourselves in the marketplace.”",
+    profile: {
+      avatar: 'https://avatars.githubusercontent.com/u/28296253?v=4',
+      name: 'Joona Rahko',
+      role: 'Staff Software Engineer',
+      company: <SvgUnity color="#007FFF" height={21} width={58} />,
+    },
+  },
+  {
+    quote:
+      "“Material UI looks great and lets us deliver fast, thanks to their solid API design and documentation - it's refreshing to use a component library where you get everything you need from their site rather than Stack Overflow. We think the upcoming version, with extra themes and customizability, will make Material UI even more of a game changer. We're extremely grateful to the team for the time and effort spent maintaining the project.”",
+    profile: {
+      avatar: 'https://avatars.githubusercontent.com/u/197016?v=4',
+      name: 'Jean-Laurent de Morlhon',
+      role: 'VP of Engineering',
+      company: undefined,
+    },
+  },
+  {
+    quote:
+      '“After much research on React component libraries, we decided to ditch our in-house library for Material UI, using its powerful customization system to implement our Design System. This simple move did a rare thing in engineering: it lowered our maintenance costs while enhancing both developer and customer experience. All of this was done without sacrificing the organization’s branding and visual identity.”',
+    profile: {
+      avatar: 'https://avatars.githubusercontent.com/u/732422?v=4',
+      name: 'Gustavo de Paula',
+      role: 'Specialist Engineer',
+      company: undefined,
```

Comment:
His company is Loggi (logo on Figma). Also, can you please change the role to `Specialist Software Engineer`?